flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/dataview/ListView.java

```diff
@@ -87,9 +84,11 @@
  *
  * }</pre>
  *
+ * <p>{@code ListView(TypeInformation<?> elementType)} method was deprecated and then removed.
+ * Please use a {@link DataTypeHint} instead.
+ *
  * @param <T> element type
  */
-@TypeInfo(ListViewTypeInfoFactory.class)
```

Comment:
I'm cucurios that we have deleted this annotation, why not delete the related deprecated classes such as `ListViewTypeInfo` and `ListViewTypeInfoFactory`?

Reply:
Thanks for reviewing the PR @lsyldliu. I have removed ListViewTypeInfoFactory as it's not used anywhere else. ListViewTypeInfo seems to be used in a few places which would need some refactoring. I think it's better to remove them in a separate PR.